### DISCUSSION TITLE
More video settings like sodium extra

### DISCUSSION THREAD

**Discussion by 0PandaDEV** [xCollateral/VulkanMod#175] [Ideas]
Could someone make an extension mod for VulkanMod, something like Sodium extra, which then adds more options in the game, such as controls for fog and weather.

## some examples

![image](https://user-images.githubusercontent.com/70103896/212156745-e7f20836-7bad-4dca-b364-b9ace682de5c.png)
![image](https://user-images.githubusercontent.com/70103896/212156758-835846b8-a10e-4def-a71a-e5b8934b6084.png)
![image](https://user-images.githubusercontent.com/70103896/212156769-d075fde1-703d-43d2-b236-62c278364f24.png)

**Comment by donnie6** [comment]
there's VulkanMod Extra. but it's not updated for newer versions.
